src/libraries/System.Private.Uri/tests/FunctionalTests/UriParameterValidationTest.cs

```diff
@@ -15,31 +15,30 @@ public class UriParameterValidationTest
         public void Uri_MakeRelativeUri_NullParameter_ThrowsArgumentException()
         {
             Assert.Throws<ArgumentNullException>(() =>
-          {
-              Uri baseUri = new Uri("http://localhost/");
-              Uri rel = baseUri.MakeRelativeUri((Uri)null);
-          });
+            {
+                Uri baseUri = new Uri("http://localhost/");
+                _ = baseUri.MakeRelativeUri((Uri)null);
+            });
         }
 
         [Fact]
         public void Uri_TryCreate_NullParameter_ReturnsFalse()
         {
             Uri baseUri = new Uri("http://localhost/");
-            Uri result;
-            Assert.False(Uri.TryCreate(baseUri, (Uri)null, out result));
-            Assert.False(Uri.TryCreate((Uri)null, baseUri, out result));
-            Assert.False(Uri.TryCreate((Uri)null, (Uri)null, out result));
+            Assert.False(Uri.TryCreate(baseUri, (Uri)null, out _));
+            Assert.False(Uri.TryCreate((Uri)null, baseUri, out _));
+            Assert.False(Uri.TryCreate((Uri)null, (Uri)null, out _)) ;
         }
 
         [Fact]
         public void Uri_IsBaseOf_NullParameter_ThrowsArgumentException()
         {
             Assert.Throws<ArgumentNullException>(() =>
-          {
-              Uri baseUri = new Uri("http://localhost/");
-              Uri relUri = null;
-              bool success = baseUri.IsBaseOf(relUri);
-          });
+            {
+                Uri baseUri = new Uri("http://localhost/");
+                Uri relUri = null;
+                _ = baseUri.IsBaseOf(relUri);
```

Comment:
Why define a local rather than just passing in null?

Reply:
Braindead pattern matching when skimming through the code, will address